x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/types.ts

```diff
@@ -69,5 +69,5 @@ export interface SearchSettings {
   boosts: Record<string, Boost[]>;
   search_fields: Record<string, SearchField>;
   result_fields?: object;
-  precision?: number;
+  precision: number;
```

Comment:
I assumed that this prop will now always be available from the API, now realizing that may have been incorrect.  @afoucret what is the case?